stripper/ze_ancient_wrath_v2_test27_m5.cfg

```diff
@@ -1,3 +1,21 @@
+;increase extreme 1 laser boss HP
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "boss_quad_encounter_01_initiate"
+	}
+	delete:
+	{
+		"OnSpawn" "boss_quad_encounter_01_hitboxRunScriptCodeStart(MAP_STAGE < MAP_MAX_STAGE ? 30 : 70, 10);2-1"
+	}
+	insert:
+	{
+		"OnSpawn" "boss_quad_encounter_01_hitboxRunScriptCodeStart(MAP_STAGE < MAP_MAX_STAGE ? 50 : 70, 10);2-1"
+	}
+}
```

Comment:
What's the justification for this?

Reply:
When this was tested , NiceShot saw how fast we killed the boss by throwing grenades where it only shot 1 laser. This was a change he made to keep it the same as CSS.